web-common/src/features/dashboards/time-series/TimeSeriesChartContainer.svelte

```diff
@@ -7,6 +7,7 @@ A container GraphicContext for the time series in a metrics dashboard.
   export let start: Date;
   export let end: Date;
   export let workspaceWidth: number;
+  export let enableFullWidth = false;
```

Comment:
I don't see this prop overridden anywhere, but I assume it's for the upcoming feature?

Reply:
Yes, it's for TDD view feature. Was prototyping it before dimension comparison. Didn't wanted to lose the additional code